tests/lib/rules-preprocessor/gjs-gts-processor-test.js

```diff
@@ -79,16 +79,17 @@ const valid = [
       </template>
     `,
   },
-  {
-    filename: 'my-component.gjs',
-    code: `
-      const Foo = <template>hi</template>
-
-      <template>
-        <Foo />
-      </template>
-    `,
-  },
+  // This test fails on master with "Preprocessing error: Cannot overwrite a zero-length range – use appendLeft or prependRight instead"
```

Comment:
@NullVoxPopuli @gitKrystan we need to get master unblocked. Can one of you submit a PR to fix or disable this test? I can't accept the current PR as-is because it has untested changes in it. It may be best to fix the test on master first and then work on the bug fix afterward.

```
FAIL tests/lib/rules-preprocessor/gjs-gts-processor-test.js
  ● template-vars › valid › 
      const Foo = <template>hi</template>

      <template>
        <Foo />
      </template>
    

    expect(received).toBe(expected) // Object.is equality

    Expected: ""
    Received: "Preprocessing error: Cannot overwrite a zero-length range – use appendLeft or prependRight instead"
```

Reply:
I'm not sure I have enough context to get this to a mergeable state, but I'd be happy to pair with someone on it.

Reply:
One of these commits is causing the error: https://github.com/ember-template-imports/ember-template-imports/compare/v3.1.1...v3.1.2

I'm about to put up a PR fixing the test

Reply:
@bmish because this feature is new, I'd be fine with leaving the test commented out for now -- there are a couple things with the position mapping in general that might need more in-depth looking, I think

Reply:
Skipping the test in a separate PR is fine as a temporary solution since it's blocking master.